eslint/rules/jquery-no-trim.js

```diff
@@ -29,13 +29,13 @@ module.exports = {
              * @param {Object} node - The node to check.
              */
             CallExpression: function (node) {
-                // jscs:disable requireCurlyBraces
+                /* eslint-disable curly */
```

Comment:
Would you be able to fix the code style to match the standard and remove the disable/enable annotation completely?

Reply:
sure! no big deal